src/trace-plugin-interface.js

```diff
@@ -199,8 +203,9 @@ PluginAPI.prototype.createChildSpan = function(options) {
       options.skipFrames ? options.skipFrames + 1 : 1);
     return new ChildSpan(this.agent_, childContext);
   } else {
-    this.logger_.warn(options.name + ': Attempted to create child span ' +
-      'without root');
+    var pluginName = options.pluginName || 'unknown plugin';
```

Comment:
Either that, or use `'custom trace'` instead of `'unknown plugin'`? Though that still doesn't cover the case of plugins that don't provide this field.